distribution/pom.xml

```diff
@@ -18,6 +18,7 @@
 
 
     <parent>
+        <!-- FIXME (DDF-1198) - Remove owasp profile when this pom uses the latest ddf-parent pom -->
```

Comment:
If it is in a different branch already, you should be able to just delete the commit.